### DISCUSSION TITLE
Not compatible with your version

### DISCUSSION THREAD

**Discussion by d1d4r** [eslint/rewrite#78] [Q&A]
hi there

i'm using sveltekit v4 and i wanna put my project in cloudflare pages, when i build project this error pop up

<html>
<body>

23:36:17.710 | npm ERR! engine Not compatible with your version of node/npm: @eslint/config-array@0.16.0
23:36:17.710 | npm ERR! notsup Not compatible with your version of node/npm: @eslint/config-array@0.16.0
23:36:17.710 | npm ERR! notsup Required: {"node":"^18.18.0 \|\| ^20.9.0 \|\| >=21.1.0"}
23:36:17.710 | npm ERR! notsup Actual:   {"npm":"9.6.7","node":"v18.17.1"}
23:36:17.712

<br class="Apple-interchange-newline">
</body>
</html>

how can i fix it

**Comment by nzakas** [comment]
The error message tells you what the problem is.
